pkg/processor/runtime/rpc/abstract_test.go

```diff
@@ -54,10 +59,13 @@ func newTestRuntime(parentLogger logger.Logger, configuration *runtime.Configura
 		return nil, errors.Wrap(err, "Failed to create runtime")
 	}
 
+	newTestRuntime.AbstractRuntime.ControlMessageBroker = NewRpcControlMessageBroker(nil,
+		parentLogger.GetChild("controlMessageBroker"))
```

Comment:
provide the parent logger, and do the `getChild` on the `NewRpcControlMessageBroker` itself